### PR TITLE
Feat/video pause play

### PR DESCRIPTION
**Pull request by TurnbulA**
Returned the play button back to original state 
It now controls if the video plays or pauses

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,9 +73,17 @@ <h1 class="header-title-text">
...
       <div class="header-image-box">
-        <img class="header-image" src="https://picsum.photos/1000/500" />
-        <div class="button-border">
-          <div class="play"></div>
+        <video id="header__video">
+          <source
+            src="resources/Video/Slaps Slow Motion.mp4"
+            type="video/mp4"
+          />
+          <div class="button-border" onclick="playPause()">
```
Comment (mali891) [inline index.html]: `addEventListener`

File: index.html [hunk 2]
```diff
@@ -73,9 +73,17 @@ <h1 class="header-title-text">
         </div>
       </div>
       <div class="header-image-box">
-        <img class="header-image" src="https://picsum.photos/1000/500" />
-        <div class="button-border">
-          <div class="play"></div>
+        <video id="header__video">
+          <source
+            src="resources/Video/Slaps Slow Motion.mp4"
```
Comment (mali891) [inline index.html]: change this to `resources/video/slaps-slow-motion.mp4`

File: functions.js [hunk 3]
```diff
@@ -11,12 +11,11 @@ window.addEventListener("scroll", () => {
     : nav.classList.remove("c-nav--alt");
 });
 
-const pageVideo = document.getElementById("header__video");
-
-function playPause() {
+const pageVideo = document.querySelector(".header__video");
+document.querySelector(".button-border .play").addEventListener("click", () => {
```
Comment (mali891) [inline functions.js]: Why is this `".button-border .play"` rather than just `".play"`?

File: functions.js [hunk 4]
```diff
@@ -11,12 +11,11 @@ window.addEventListener("scroll", () => {
...
   if (pageVideo.paused) {
```
Comment (mali891) [inline functions.js]: ternary?

File: functions.js (hunk 3 above)
Comment (TurnbulA) [reply functions.js]: Button border encapsulates play

**Pull request closed without merging**